Test/UnitaryDynamics/unit-timeevolution.cpp

```diff
@@ -8,9 +8,9 @@
 #include <Eigen/Eigen>
 
 #include "netket.hpp"
-#include "UnitaryTimeEvolution/time_evolution.hpp"
-#include "UnitaryTimeEvolution/TimeStepper/explicit_time_steppers.hpp"
-#include "UnitaryTimeEvolution/TimeStepper/controlled_time_steppers.hpp"
+#include "UnitaryDynamics/time_evolution.hpp"
```

Comment:
Can we change the name to `Dynamics` or just `TimeEvolution`? For the imaginary time case this is not a unitary evolution for example